packages/polymorphic/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import { toBePolymorphic } from './utils/Polymorphic.testutils';
+export { Polymorph, BasePolymorph } from './Polymorph';
+export { Polymorphic } from './Polymorphic';
+
+export {
+  usePolymorphicRef,
+  usePolymorphicComponent,
+  usePolymorphic,
+} from './Polymorphic.hooks';
+
+export type {
+  PolymorphicAs,
+  PolymorphicProps,
+  PolymorphicPropsWithRef,
+  PolymorphicRef,
+  PolymorphicComponentType,
+} from './Polymorphic.types';
+
+export { useInferredPolymorphic } from './InferredPolymorphic';
+
+export type { InferredPolymorphicProps } from './InferredPolymorphic';
+
+expect.extend({
```

Comment:
should this be exported from test utils instead, or are we intentionally making this external?

Reply:
I'm intentionally making this external. I've already written `expect(InputOption).toBePolymorphic()`

Reply:
kk